addons/danger/XEH_preInitClient.sqf

```diff
@@ -10,7 +10,7 @@ Here four buttons are added:
 These functions and buttons could conceivably be moved to their own module.
 They might also benefit from added interface and more nuanced functions handling.
 That said: within the current scope of the AI mod. They can make a comfortable
-home here. Until revisited by more capable personnell.
+home here. Until revisited by more capable personell.
```

Comment:
As far as I can find, the former `personnell` was closer, should just drop the extra trailing `l`. Still, I'd go with `people` instead, since seemingly `personnel` is people working in a company:
```suggestion
home here. Until revisited by more capable people.
```